jets.gemspec

```diff
@@ -66,4 +66,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
+  spec.add_development_dependency "cfn-status"
```

Comment:
I added it because I couldn't run the test locally without it.

Reply:
Good to know. Maybe it should be add_dependency. Wondering if it may be breaking for others.

Reply:
Will move forward and change if needed. Thanks!